modules/uncertainty/R/ensemble.R

```diff
@@ -78,28 +78,32 @@ read.ensemble.output <- function(ensemble.size, pecandir, outdir, start.year, en
 ##' @return matrix of (quasi-)random samples from trait distributions
 ##' @export
 ##' @author David LeBauer, Istem Fer
-get.ensemble.samples <- function(ensemble.size, pft.samples, env.samples, 
-                                 method = "uniform", param.names = NULL, ...) {
+get.ensemble.samples <- function(ensemble.size, pft.samples,method = "uniform",
```

Comment:
Hey, while you're fixing things, could you change this so the possible options show up in the autocomplete better?
```suggestion
get.ensemble.samples <- function(ensemble.size, pft.samples,method = c("uniform", "halton", "sobol", "torus", "lhc"),
```
Then to the beginining of the function body add `method <- match.arg(method)`